### PR TITLE
Support statfs for jnifuse

### PR DESCRIPTION
**Pull request by maobaolong**
### What changes are proposed in this pull request?

Support statfs for jnifuse
Address #14888

### Why are the changes needed?

Add statfs implementation.

### Does this PR introduce any user facing changes?

`df -i` can shows the usage of alluxio storage.

### CODE REVIEW

File: integration/jnifuse/native/src/main/native/libjnifuse/jnifuse_impls.h [hunk 1]
```diff
@@ -31,6 +31,7 @@ int getxattr_wrapper(const char *path, const char *name, char *value, size_t siz
 int setxattr_wrapper(const char *path, const char *name,
                      const char *value, size_t size, int flags);
 #endif
+int statfs_wrapper(const char* path, struct statvfs* stbuf);
```
Comment (apc999) [inline integration/jnifuse/native/src/main/native/libjnifuse/jnifuse_impls.h:34]: put it in alphabetical order?

**Review by apc999 — APPROVED** [review]
LGTM with minor comments left

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java [hunk 2]
```diff
@@ -972,6 +981,59 @@ public int symlink(String linkname, String path) {
...
+      BlockMasterInfo blockMasterInfo = blockClient.getBlockMasterInfo(blockMasterInfoFilter);
+
+      // although user may set different block size for different files,
+      // small block size can result more accurate compute.
+      long blockSize = 4L * Constants.KB;
+      // fs block size
+      // The size in bytes of the minimum unit of allocation on this file system
+      stbuf.f_bsize.set(blockSize);
+      // The preferred length of I/O requests for files on this file system.
+      stbuf.f_frsize.set(blockSize);
```
Comment (yyongycy) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java:1017]: How could this size so small? Just 4K?

Comment (LuQQiu) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java:1017]: will this affect the fuse.read(size)? will a bigger value resulting in fuse.read() with a larger size?

Comment (yyongycy) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java:1017]: Some app will set the buffer size accordingly. This flag is a heuristic one.

**Review by yyongycy — APPROVED** [review]

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by Xenorith** [comment]
reminder to declare any failing unit tests before merging:
`alluxio.client.fs.io.FileOutStreamIntegrationTest.cancelWrite`

**Pull request merged**